modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/service/test/CommerceOrderItemLocalServiceTest.java

```diff
@@ -983,6 +985,68 @@ public void testDeleteProductBundleWithOptionWithSKU() throws Exception {
 		}
 	}
 
+	@Test
+	public void testImportCommerceOrderItem() throws Exception {
+		CPInstance cpInstance = CPTestUtil.addCPInstance(_group.getGroupId());
```

Comment:
I don't know about this order - what about
* commerceInventoryWarehouse
* addWarehouseCommerceChannelRel
* cpInstance
* commerceInventoryWarehouseItems
* cpDefinition
* cpOption
* cpDefinitionOptionRel
* order
* orderItem
?